src/components/KonfluxSystemNotifications/__tests__/NotificationItem.spec.tsx

```diff
@@ -29,7 +29,9 @@ describe('NotificationItem', () => {
 
   it('renders timestamp using Timestamp component', () => {
     render(<NotificationItem {...validDangerNotification} />);
-    expect(screen.getByText('Aug 10, 2025, 11:08 AM')).toBeInTheDocument();
+    // The timestamp will be formatted according to the user's locale and timezone
```

Comment:
on some systems it can occur that you will have 10.08.2025. I have similair issue in already existing tests.